allennlp/interpret/saliency_interpreters/simple_gradient.py

```diff
@@ -48,9 +46,9 @@ def saliency_interpret_from_json(self, inputs: JsonDict) -> JsonDict:
                 # gradient and its respective embedding.
                 input_idx = int(key[-1]) - 1
                 # The [0] here is undo-ing the batching that happens in get_gradients.
-                emb_grad = numpy.sum(grad[0] * embeddings_list[input_idx][0], axis=1)
```

Comment:
This line is computing a dot product between the gradient vector and the embedding vector.  The original implementation is correct.  The proposed change is not a dot product anymore.